packages/shared/src/components/auth/OnboardingRegistrationForm4d5.tsx

```diff
@@ -43,6 +44,7 @@ export const OnboardingRegistrationForm4d5 = ({
   trigger,
   className,
   onClose,
+  onShowLoginOptions,
 }: OnboardingRegistrationForm4d5Props): ReactElement => {
   const { trackEvent } = useContext(AnalyticsContext);
   const [shouldLogin, setShouldLogin] = useState(false);
```

Comment:
Is this one even used here now then?

Reply:
Yeah, it's set to true of signing up with email and it exists, it shows an error message.

Reply:
Oh ok, but not the actual screen, more asking from the event point of view.

Reply:
Since this screen is only signups, it should only send signup events. Once you click the login button, you're navigated to a new screen which would then send a login even
![image](https://github.com/dailydotdev/apps/assets/1681525/2e36a6ac-d40e-47d0-b8d5-2f933c0d3b77)
![image](https://github.com/dailydotdev/apps/assets/1681525/833fb780-05ae-41b0-9608-ef280a8d39b9)